packages/plugin-react/src/index.ts

```diff
@@ -18,18 +12,9 @@ import {
   runtimePublicPath,
 } from './fast-refresh'
 
-// lazy load babel since it's not used during build if plugins are not used
-let babel: typeof babelCore | undefined
-async function loadBabel() {
-  if (!babel) {
-    babel = await import('@babel/core')
-  }
-  return babel
-}
-
 export interface Options {
-  include?: string | RegExp | Array<string | RegExp>
-  exclude?: string | RegExp | Array<string | RegExp>
+  jsxInclude?: string | RegExp | string[] | RegExp[]
```

Comment:
Does it mean you can't have something like this with oxc?

```ts
include: ['file', /some-file/]
```

Reply:
The change is out-date, you could using @vitejs/plugin-react-oxc , it already migrate to https://github.com/vitejs/vite-plugin-react/pull/439.